backend/pokemon/tests.py

```diff
@@ -1,4 +1,86 @@
-# from django.test import TestCase
+from django.test import TestCase
 
+from model_bakery import baker
 
-# Create your tests here.
+from pokemon.helpers import is_team_valid
+
+
+class PokemonSumTest(TestCase):
+    def setUp(self):
+        self.pokemon_1 = baker.make(
+            "pokemon.Pokemon",
+            poke_id=15,
+            attack=80,
+            defense=45,
+            hp=75,
+        )
+        self.pokemon_2 = baker.make(
+            "pokemon.Pokemon",
+            poke_id=16,
+            attack=35,
+            defense=35,
+            hp=56,
+        )
+        self.pokemon_3 = baker.make(
+            "pokemon.Pokemon",
+            poke_id=17,
+            attack=50,
+            defense=50,
+            hp=71,
+        )
+        self.pokemon_4 = baker.make(
+            "pokemon.Pokemon",
+            poke_id=18,
+            attack=70,
+            defense=70,
+            hp=101,
+        )
+
+        self.pokemon_1.save()
+        self.pokemon_2.save()
+        self.pokemon_3.save()
+        self.pokemon_4.save()
+
+    def test_team_sum(self):
```

Comment:
I think this name could be improved, like `test_if_team_is_valid_if_sum_is...`. We can also use test as documentation